src/test/java/seedu/address/logic/parser/EditCommandParserTest.java

```diff
@@ -94,12 +94,6 @@ public void parse_invalidValue_failure() {
         // is tested at {@code parse_invalidValueFollowedByValidValue_success()}
         assertParseFailure(parser, " i/1" + PHONE_DESC_BOB + INVALID_PHONE_DESC, Phone.MESSAGE_CONSTRAINTS);
 
-        // while parsing {@code PREFIX_TAG} alone will reset the tags of the {@code Client} being edited,
```

Comment:
seems like this just needs a refactor, not complete removal of the test, do lmk the rationale behind removing these!

Reply:
this will be fixed in v1.4